tests/toolchain/gcc5_C_compilation.pm

```diff
@@ -0,0 +1,35 @@
+use base "opensusebasetest";
+use strict;
+use testapi;
+
+sub run() {
+    my $self = shift;
+
+    type_string "wget https://github.com/linux-test-project/ltp/releases/download/20150420/ltp-full-20150420.tar.bz2 && echo 'OK'\n";
+    type_string "tar jxf ltp-full-20150420.tar.bz2\n";
+    sleep 2;
+    type_string "export CC=/usr/bin/gcc-5\n";
+    type_string "export CXX=/usr/bin/g++-5\n";
+    type_string "cd ltp-full-20150420\n";
+    sleep 2;
+    type_string "./configure --with-open-posix-testsuite|tee /dev/$serialdev && echo 'configure PASSED'|tee /dev/$serialdev\n";
```

Comment:
Ah, now I see
I forgot this tee there, I just used it for debugging the compilation